rust/platform/channel/BUILD

```diff
@@ -0,0 +1,22 @@
+package(default_visibility = ["//visibility:public"])
+
+# Allows toolchains and targets to constrain themselves based on Rust release
+# channels, for example when depending on features only available in Nightly.
+#
+# https://doc.rust-lang.org/book/appendix-07-nightly-rust.html
+constraint_setting(name = "channel")
```

Comment:
I assume this is not a global name, right? I am not familiar w/ defining constraints.

Reply:
This is a target name, it's referenced as `@io_bazel_rules_rust//rust/platform/channel` and the constraint values are referenced like `@io_bazel_rules_rust//rust/platform/channel:nightly`.